release.mk

```diff
@@ -50,8 +50,13 @@ endif
 
 # BASE_BRANCH select by release type (default patch)
 ifeq ($(RELEASE_TYPE),minor)
-	BASE_BRANCH ?= main
-	CHANGELOG_BRANCH = main
+	ifeq ($(PROJECT_MAJOR_VERSION), 8)
```

Comment:
I think this is not possible, `8.19` will only contain patch releases

Reply:
`8.19.0` is still to be released though. But I ended up reverting this commit as the automation was only partially working.